var/spack/repos/builtin/packages/gdbm/package.py

```diff
@@ -16,6 +16,7 @@ class Gdbm(AutotoolsPackage, GNUMirrorPackage):
     homepage = "http://www.gnu.org.ua/software/gdbm/gdbm.html"
     gnu_mirror_path = "gdbm/gdbm-1.13.tar.gz"
 
+    version('1.19.1', sha256='37ed12214122b972e18a0d94995039e57748191939ef74115b1d41d8811364bc')
```

Comment:
This has the wrong sha and is breaking build tests, since it is installed as part of them.

Reply:
Also, I don't seem to be able to fetch this version @robert-mijakovic ?

Reply:
Strange, I'll have a look  ASAP.